src/css.js

```diff
@@ -255,7 +263,7 @@ jQuery.extend( {
 			if ( !hooks || !( "set" in hooks ) ||
 				( value = hooks.set( elem, value, extra ) ) !== undefined ) {
 
-				style[ name ] = value;
+				customProperty ? style.setProperty( name, value ) : style[ name ] = value;
```

Comment:
Okay, so i have found three related rules in ESLint -

1. `no-nested-ternary` - http://eslint.org/docs/rules/no-nested-ternary
1. `no-ternary` - http://eslint.org/docs/rules/no-ternary
1. `no-unneeded-ternary` - http://eslint.org/docs/rules/no-unneeded-ternary

It seems those rules wouldn't satisfy us, but it seems `no-nested-ternary` would be a good fit? Created - https://github.com/jquery/eslint-config-jquery/issues/2

Didn't found any similar proposals in ESLint issues, will bring this up in ESLint gitter.